src/components.jl

```diff
@@ -157,7 +161,7 @@ function translate_connections(doc::Document, systems, class)
         sys2 = systems[c2]
         for w in variables_of.(list_connection_variables(k))
             v1, v2 = Symbol.(w)
-            if class[make_var(c1,v1)] && class[make_var(c2,v2)] && Symbol(sys1.iv) != v1
```

Comment:
was this supposed to be deleted?

Reply:
> You can remove this from the condition (it will be removed in the new version anyway).

yeah @shahriariravanian asked me to in slack

Reply:
I don't think any of this code is actually used yet. It's WIP for components